File: GoLang/model/Applicant.go
package model

type Applicant struct {
	ID             string `json:"id,omitempty"`
	CreatedAt      string `json:"createdAt,omitempty"`
	Key            string `json:"key,omitempty"`
	ClientID       string `json:"clientId,omitempty"`
	InspectionID   string `json:"inspectionId,omitempty"`
	ExternalUserID string `json:"externalUserId,omitempty"`
	Info           Info   `json:"info,omitempty"`
	FixedInfo      Info   `json:"fixedInfo,omitempty"`
	Review         struct {
		ElapsedSincePendingMs int    `json:"elapsedSincePendingMs,omitempty"`
		ElapsedSinceQueuedMs  int    `json:"elapsedSinceQueuedMs,omitempty"`
		Reprocessing          bool   `json:"reprocessing,omitempty"`
		CreateDate            string `json:"createDate,omitempty"`
		ReviewDate            string `json:"reviewDate,omitempty"`
		StartDate             string `json:"startDate,omitempty"`
		ReviewResult          struct {
			ReviewAnswer string `json:"reviewAnswer,omitempty"`
		} `json:"reviewResult,omitempty"`
		ReviewStatus           string `json:"reviewStatus,omitempty"`
		NotificationFailureCnt int    `json:"notificationFailureCnt,omitempty"`
		Priority               int    `json:"priority,omitempty"`
	} `json:"review,omitempty"`
	Lang string `json:"lang,omitempty"`
	Type string `json:"type,omitempty"`
}

File: C#/AppTokenCSharpExample/DataModel/Applicant.cs
namespace AppTokenCSharpExample.DataModel;

public class Applicant
{
    public string id { get; set; }
    public string createdAt { get; set; }
    public string clientId { get; set; }
    public string inspectionId { get; set; }
    public string externalUserId { get; set; }
    public Fixedinfo fixedInfo { get; set; }
    public string email { get; set; }
    public string phone { get; set; }
    public Requirediddocs requiredIdDocs { get; set; }
    public Review review { get; set; }
    public string type { get; set; }
}